src/Stratis.Features.FederatedPeg.Tests/CrossChainTransferStoreTests.cs

```diff
@@ -469,6 +469,30 @@ public void StoredPartialTransactionsTriggerSignatureRequest()
             }
         }
 
+        [Fact]
+        public void NextMatureDepositStartsHigherOnMain()
+        {
+            // This should really be 2 tests in separate classes but we'll fit in with what is already happening for now.
+
+            // Start querying counter-chain for deposits from first non-genesis block on main chain and a higher number on side chain.
+            int depositHeight = (this.network.Name == new StratisRegTest().Name)
```

Comment:
Perhaps add a variable to the network class: `FirstSidechainDepositHeight`.

Reply:
That would be nice, but it would be making changes to the StratisMain network for everyone, when only the sidechain nodes need to know about it.

Wonder if there's a way to store it somewhere else useful